examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -0,0 +1,378 @@
+
+    if data_args.audio_column_name not in ds["train"].column_names:
+        raise ValueError(
+            f"--audio_column_name {data_args.audio_column_name} not found in dataset '{data_args.dataset_name}'. "
+            "Make sure to set `--audio_column_name` to the correct audio column - one of "
+            f"{', '.join(ds['train'].column_names)}."
+        )
+
+    if data_args.label_column_name not in ds["train"].column_names:
+        raise ValueError(
+            f"--label_column_name {data_args.label_column_name} not found in dataset '{data_args.dataset_name}'. "
+            "Make sure to set `--label_column_name` to the correct text column - one of "
+            f"{', '.join(ds['train'].column_names)}."
+        )
+
+    feature_extractor = Wav2Vec2FeatureExtractor.from_pretrained(
+        model_args.feature_extractor_name or model_args.model_name_or_path,
+        return_attention_mask=True,
+        cache_dir=model_args.cache_dir,
+        revision=model_args.model_revision,
+        use_auth_token=True if model_args.use_auth_token else None,
+    )
+
+    def train_transforms(batch):
+        """Apply train_transforms across a batch."""
+        output_batch = {"input_values": []}
+        for f in batch[data_args.audio_column_name]:
+            wav = load_audio(f, sample_rate=feature_extractor.sampling_rate)
+            wav = random_subsample(
+                wav, max_length=data_args.max_length_seconds, sample_rate=feature_extractor.sampling_rate
+            )
+            output_batch["input_values"].append(wav)
+        output_batch["labels"] = [label for label in batch[data_args.label_column_name]]
+
+        return output_batch
+
+    def val_transforms(batch):
+        """Apply val_transforms across a batch."""
+        output_batch = {"input_values": []}
+        for f in batch[data_args.audio_column_name]:
+            wav = load_audio(f, sample_rate=feature_extractor.sampling_rate)
+            output_batch["input_values"].append(wav)
+        output_batch["labels"] = [label for label in batch[data_args.label_column_name]]
+
+        return output_batch
+
+    # Prepare label mappings.
+    # We'll include these in the model's config to get human readable labels in the Inference API.
+    labels = ds["train"].features[data_args.label_column_name].names
+    label2id, id2label = dict(), dict()
+    for i, label in enumerate(labels):
+        label2id[label] = str(i)
+        id2label[str(i)] = label
+
+    # Load the accuracy metric from the datasets package
+    metric = datasets.load_metric("accuracy")
+
+    # Define our compute_metrics function. It takes an `EvalPrediction` object (a namedtuple with a
+    # predictions and label_ids field) and has to return a dictionary string to float.
+    def compute_metrics(p):
+        """Computes accuracy on a batch of predictions"""
+        return metric.compute(predictions=np.argmax(p.predictions, axis=1), references=p.label_ids)
```

Comment:
```suggestion
        predictions = np.argmax(p.predictions, axis=1)
        return metric.compute(predictions=predictions, references=p.label_ids)
```
Let's split the argmax in another line for easier reading.